src/Administration/Resources/app/administration/src/app/snippet/de.json

```
@@ -730,7 +730,8 @@
         "promotionsInCartCountRule": "Anzahl der Rabatte",
         "lineItemUnitPriceRule": "Positionsstückpreis",
         "lineItemTotalPriceRule": "Positionszwischensumme",
-        "lineItemWithQuantityRule": "Positionsanzahl",
+        "lineItemPerItemQuantityRule": "Positionsanzahl",
+        "lineItemWithQuantityRule": "Anzahl einer bestimmten Position",
```

feels still confusing... 
what do you think about:
```suggestion
        "lineItemPerItemQuantityRule": "Anzahl je Position",
        "lineItemWithQuantityRule": "Anzahl einer bestimmten Position",
```